<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidImageBinarizerClass</class>
 <widget class="QMainWindow" name="WidImageBinarizerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1437</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Binarizer Demo App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="m_pushButtonOpenImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Open image...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pushButtonBinarize">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>269</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Binarize AND detect text</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>371</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Source image</string>
    </property>
    <widget class="QLabel" name="m_labelImageSrc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="m_labelSauvolaRange">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Souvola range</string>
    </property>
   </widget>
   <widget class="QSlider" name="m_sliderSauvolaRange">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="m_labelSauvolaFactor">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Souvola factor</string>
    </property>
   </widget>
   <widget class="QSlider" name="m_sliderSauvolaFactor">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="m_tabWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>961</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Binarization function</string>
    </property>
    <widget class="QRadioButton" name="m_radioFuncSauvola">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sauvola</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radioFuncLeptonica">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>From Leptonica</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radioFuncNone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radioFuncSauvolaFast">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sauvola fast</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_lineEditStatus">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>770</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>560</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Result render options</string>
    </property>
    <widget class="QCheckBox" name="m_checkBoxRenderBinarizedImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Render binaried image</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="m_checkBoxRenderRecogniedText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Render recognized text</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="m_checkBoxRenderRectangles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Render rectangles</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>680</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Edit actions</string>
    </property>
    <widget class="QPushButton" name="m_buttonRemoveSelectedRectangles">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove selected rectangles</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="m_buttonDown">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../res/WidImageBinarizer.qrc">
      <normaloff>:/arrow_down.png</normaloff>:/arrow_down.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="m_buttonUp">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../res/WidImageBinarizer.qrc">
      <normaloff>:/arrow_up.png</normaloff>:/arrow_up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_linePage">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pdf page</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_lineScale">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pdf scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_buttonPlus">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../res/WidImageBinarizer.qrc">
      <normaloff>:/arrow_plus.png</normaloff>:/arrow_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="m_buttonMinus">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../res/WidImageBinarizer.qrc">
      <normaloff>:/arrow_minus.png</normaloff>:/arrow_minus.png</iconset>
    </property>
   </widget>
   <widget class="QProgressBar" name="m_progressRecognition">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>770</y>
      <width>991</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>pdf rotate</string>
    </property>
    <widget class="QRadioButton" name="m_radioRotateNone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>none</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radioRotateLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>left</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radioRotateRight">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>right</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="m_buttonCompareBinarized">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Comapre binarized</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1437</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/WidImageBinarizer.qrc"/>
 </resources>
 <connections/>
</ui>
